frontend/src/app/components/ExperimentTourDialog/index.js

```diff
@@ -6,44 +6,85 @@ import React from "react";
 
 import StepModal from "../StepModal";
 
+import type { InstalledExperiments } from "../../reducers/addon";
+
+import { getBreakpoint } from "../../containers/App";
 import { experimentL10nId } from "../../lib/utils";
 
 type ExperimentTourDialogProps = {
+  hasAddon?: boolean,
+  installed?: InstalledExperiments,
   experiment: Object,
   isExperimentEnabled: Function,
   onCancel: Function,
   onComplete: Function,
-  sendToGA: Function
+  sendToGA: Function,
+  isFeatured?: boolean
 }
 
 export default class ExperimentTourDialog extends React.Component {
   props: ExperimentTourDialogProps
 
+  applyAdditionalMetricsArgs(gaArgs: Object) {
+    const { experiment, installed, hasAddon } = this.props;
+    return Object.assign(gaArgs, {
+      dimension1: hasAddon,
+      dimension2: installed ? Object.keys(installed).length > 0 : false,
+      dimension3: installed ? Object.keys(installed).length : 0,
+      dimension10: getBreakpoint(window.innerWidth),
```

Comment:
`sendToGA` [already includes](https://github.com/mozilla/testpilot/blob/ee2d65de75f721022b77ee1c06fbd95b412bf7ae/frontend/src/app/containers/App/index.js#L249) `dimension10` for all requests.